### ISSUE TITLE
Feature Request: Change playback speed

### ISSUE THREAD

**Issue by JakeDluhy** [spotify/web-playback-sdk#46]
I'm interested in using spotify to practice music and dancing. I've already used the web playback sdk to be able to loop a specific portion of a song over and over, but it would be really awesome to be able to slow down the song to 0.5, 0.75, and 0.9 tempos.

I'm pretty sure this is possible through javascript on the html audio element, and so I would think (hopefully) that it wouldn't be too hard to expose it publicly.

Thanks!

**Comment by beebase** [comment]
Yes, would be awesome to change the speed like the playbackRate in the html audio tag.
Then we could play entire song lists in one tempo.

**Comment by tobika** [comment]
This sounds a bit hacky but seems to work :-)
https://github.com/intOrfloat/spotitySpeedExtension

But I have no idea if this breaks things on track change or other events.

Let's hope this functionality will make it to the web sdk at some point  :+1:
